index.html

```diff
@@ -135,10 +135,66 @@ <h1>Multi-Post <br> Stories</h1>
           <li class="ruby_tag">Ruby on rails</li>
           <li>javaScript</li>
         </ul>
-        <button>See Project</button>
+        <button onclick="showPopup(3)">See Project</button>
```

Comment:
- [ ] Please kindly ensure that the projects section of the page is generated dynamically using Javascript. Please remove all HTML content on the project section and generate with Javascript.